dev-tools/packaging/templates/docker/Dockerfile.elastic-agent.tmpl

```diff
@@ -144,8 +143,10 @@ true
 RUN mkdir /app
 {{- end }}
 {{- else }}
-RUN groupadd --gid 1000 {{ .BeatName }}
-RUN useradd -M --uid 1000 --gid 1000 --groups 0 --home {{ $beatHome }} {{ .user }}
+RUN groupadd --gid 1000 {{ .BeatName }} && \
+    useradd -M --uid 1000 --gid 1000 --groups 0 --home {{ $beatHome }} {{ .user }} && \
+    chown -R {{ .user }}:{{ .user }} {{ $beatHome }} && \
+    true
```

Comment:
Why do you need a `&& true` at the end ? :thinking:

Reply:
convention & readability, easier to add other commands later without messing with `&& \` at the end